src/hooks/useRebalanceHandler.tsx

```diff
@@ -21,6 +29,10 @@ export const useRebalanceHandler = () => {
   const [slippage, setSlippage] = useState<string>('')
   const [slippageMinutes, setSlippageMinutes] = useState<string>('')
 
+  const [posManager, setPosManager] = useState<Contract | null>(null)
```

Comment:
you can default this value to line 12

Reply:
This is of contract type so we can not add string to it.